test/framework/src/main/java/org/elasticsearch/test/InternalTestCluster.java

```diff
@@ -220,6 +227,7 @@
     private Function<Client, Client> clientWrapper;
 
     public InternalTestCluster(String nodeMode, long clusterSeed, Path baseDir,
+                               int numMasterNodes,
```

Comment:
do we actually need to set something > 0 from the outside? Or do we end up always using either -1 (meaning do randomize) or 0 (no dedicated masters)?